_schema.graphql

```diff
@@ -103,6 +103,14 @@ type AnalyticsPageviewStats {
   uniqueVisitors: Int
 }
 
+# Inquiry stats of a partner
+type AnalyticsPartnerInquiryStats {
+  inquiryCount: Int!
+  inquiryResponseTime: Int
```

Comment:
Hmm so inquiry count is non-nullable but inquiry response time is - should we make those consistent? I'm not sure why we'd prefer one over the other off the top of my head, or why they might be different

Reply:
Good catch. `AnalyticsPartnerSalesStats` has both of these fields typed as non-nullable so I think we should also be consistent across both types. I am leaning towards making `inquiryResponseTime` non-nullable since it will involve the least amount of changes.

Reply:
I spoke with @sepans and it confirmed that this field was intentionally left nullable. Doing so allows clients the option of taking an action if the response time is `nil` - in the case of CMS analytics dashbaord that means we have the option of not displaying a breakdown item for inquiry response times.

We have a ticket in our backlog to further explore this issue: https://artsyproduct.atlassian.net/browse/GALL-1783

Reply:
ah that's right, thanks for investigating @iskounen. Let's leave as-is!